ot-node.js

```diff
@@ -137,19 +137,30 @@ class OTNode {
         }
     }
 
-    async createProfile() {
+    async createProfiles() {
         const blockchainModuleManager = this.container.resolve('blockchainModuleManager');
-        if (!blockchainModuleManager.identityExists()) {
-            const networkModuleManager = this.container.resolve('networkModuleManager');
-            const peerId = networkModuleManager.getPeerId();
-            await blockchainModuleManager.deployIdentity();
-            await blockchainModuleManager.createProfile(peerId);
-            if (process.env.NODE_ENV !== 'development' && process.env.NODE_ENV !== 'test') {
-                this.saveIdentityInUserConfigurationFile(blockchainModuleManager.getIdentity());
+        const createProfilesPromises = Object.keys(
+            this.config.modules.blockchain.implementation,
```

Comment:
Add method in blockchainModuleManager getAllImplementationNames - something like this and use it here